sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2262,6 +2246,22 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
+   * Returns the current date.
+   *
+   * @group datetime_funcs
+   * @since 1.5.0
+   */
+  def current_date(): Column = CurrentDate()
+
+  /**
+   * Returns the current timestamp.
+   *
+   * @group datetime_funcs
+   * @since 1.5.0
+   */
+  def current_timestamp(): Column = CurrentTimestamp()
```

Comment:
This is not like the case of e/pi, there were some considerations in the previous pr for this.
Anyway, this is just a movement.